app/views/idv/session_errors/recovery_warning.html.erb

```diff
@@ -1,22 +0,0 @@
-<%= render(
```

Comment:
There's two more of these, `recovery_failure.html.erb` and `recovery_throttled.html.erb`, which I'd expect can be removed as well, along with their specs.